test-infra/src/main/java/com/datastax/oss/driver/api/testinfra/ccm/CcmBridge.java

```diff
@@ -52,7 +52,7 @@ public class CcmBridge implements AutoCloseable {
   private static final Logger LOG = LoggerFactory.getLogger(CcmBridge.class);
 
   public static final Version VERSION =
-      Objects.requireNonNull(Version.parse(System.getProperty("ccm.version", "4.0.0")));
+      Objects.requireNonNull(Version.parse(System.getProperty("ccm.version", "4.0.10")));
```

Comment:
Bringing default Cassandra version used in testing to be same 4.0 one used by Jenkins.

This is why this test passed locally but not in 4.0/3.11 in Jenkins.

Happy to remove if this change is too disruptive.

Reply:
Since we're talking about a default here I'd prefer to avoid changing it for now.  Worth noting that on Jenkins at least this value is always provided [by the runner](https://github.com/riptano/openstack-jenkins-drivers/blob/master/shared/scripts/ccm_environment.sh#L260) by way of the [Jenkinsfile](https://github.com/datastax/java-driver/blob/4.x/Jenkinsfile#LL101C1-L101C1).  So for our purposes it will always be set to the expected version based on Jenkins args.

Reply:
Ack, removed